extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -320,8 +321,8 @@ amp-story-page.i-amphtml-story-page-shown {
 }
 
 [desktop] .i-amphtml-story-share .i-amphtml-story-share-list > li {
-  width: 56px;
-  height: 56px;
+  width: 56px!important;
+  height: 56px!important;
```

Comment:
Nit: Looks like use of space before `!important` is inconsistent.

Reply:
i dont really see any space before `!important` in the rest of the file... is it a required thing?

Reply:
The rest of AMP code prefixes with a single space. Not critical but please fix moving forward.